tests/test_nonlinearity.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+from vintch_model.backend_config import (
+    TorchBackend,
+    JaxBackend,
+    NumpyBackend,
+)
+from vintch_model.nonlinearity import TentNonlinearity
+
+
+class TestNonlinearity:
+    @pytest.mark.parametrize(
+        "backend_cls",
+        [
+            TorchBackend,
+            JaxBackend,
+            NumpyBackend,
+        ],
+    )
+    @pytest.mark.parametrize(
+        "nonlinearity_mode",
+        ["relu", "quadratic"],
+    )
+    def test_reshapes(self, backend_cls, nonlinearity_mode):
+        backend = backend_cls()
+        non_linearity = TentNonlinearity(
+            backend_instance=backend, nonlinearity_mode=nonlinearity_mode
+        )
+        x = non_linearity._tent_centers
```

Comment:
why is this the input? nonlinearity should work with arbitrarily shaped inputs, right? I'd try many different shapes.